src/environments/environment.prod.ts

```diff
@@ -2,26 +2,27 @@ import { commonEnv } from './environment.common';
 
 const env: Partial<typeof commonEnv> = {
   production: true,
-  environmentName: "production",
+  environmentName: 'production',
   featureVisible: true,
   application: true,
-  theme: "default",
-  rpcUrl: "https://consortia-rpc.energyweb.org/",
+  theme: 'default',
+  rpcUrl: 'https://consortia-rpc.energyweb.org/',
   chainId: 246,
   cacheServerUrl: "https://identitycache.energyweb.org/v1",
   natsServerUrl: "https://identityevents.energyweb.org/",
   kmsServerUrl: "https://kms.energyweb.org/connect/new",
-  stakingPoolFactoryAddress: "0xc11aa8A6Ca3de02c7eB106Fb8e85cee051c7B2Fe",
+  stakingPoolFactoryAddress: "0x181A8b2a5AEb25941F6A79b4aE43dBb1968c417A",
```

Comment:
Are you sure the staking contract address should be changed? This seems like the right one, but I'm confused about how it was working before. Does this change make sense to you @dawidgil ?

Reply:
just develop branch wasn't updated. Now develop should be adjusted to the master branch (had conflict earlier).

Reply:
and not master but release branch 😄

Reply:
So the `stakingPoolFactoryAddress` was incorrect in `environment.prod.ts` on the `release` branch previously? And now it will be the correct address?

Reply:
no, it was incorrect on develop branch.